tap_outreach/sync.py

```diff
@@ -327,14 +322,14 @@ def update_current_stream(state, stream_name=None):
     singer.write_state(state)
 
 
-def sync(client, config, catalog, state, start_date):
+def sync_stream(client, config, catalog, state, start_date):
```

Comment:
What is the main intension of renaming this method? We use same convention in other taps as well, so IMO we don't need this change unless I am missing something.

Reply:
referred sync function in __init__.py as _sync to avoid the confusion.